lib/foodcritic/version.rb

```diff
@@ -1,4 +1,4 @@
 module FoodCritic
   # The current version of foodcritic
-  VERSION = '6.0.1'
+  VERSION = '6.0.2'
```

Comment:
Can you remove this.  We'll handle the version bump when we're ready for a release